pycds/database.py

```diff
@@ -5,7 +5,7 @@
 
 
 def check_migration_version(
-    executor, schema_name=get_schema_name(), version="bb2a222a1d4a"
+    executor, schema_name=get_schema_name(), version="fecff1a73d7e"
```

Comment:
I wonder if this can and should be done with `pycds.alembic.info.get_head_revision()`